test-utils/runtime/src/lib.rs

```diff
@@ -176,6 +176,19 @@ impl serde::Serialize for Extrinsic {
 	}
 }
 
+// TODO: rustc can't deduce this trait bound https://github.com/rust-lang/rust/issues/48214
```

Comment:
TODO what?

Reply:
nothing :D but I think there is some anonymous lifetime that forced this